controller/RestEligibilities.php

```diff
@@ -144,7 +148,27 @@ private function getDeliveryFromRequest()
         } catch (common_exception_MissingParameter $e) {
             throw new common_exception_RestApi(__('Missed required parameter: `%s`', self::PARAMETER_DELIVERY_ID));
         } catch (common_exception_NotFound $e) {
-            throw new common_exception_RestApi("Delivery `{$deliveryUri}` does not exist.");
+            throw new common_exception_NotFound(__('Delivery `%s` does not exist.', $deliveryUri));
         }
     }
+
+    /**
+     * @param $testCenter
+     * @param $delivery
+     * @return Eligibility
+     * @throws \common_exception_InconsistentData
+     * @throws common_exception_NotFound
+     */
+    private function getEligibilityByTestCenterAndDelivery($testCenter, $delivery)
+    {
+        /** @var EligibilityService $eligibilityService */
+        $eligibilityService = $this->getServiceLocator()->get(EligibilityService::SERVICE_ID);
+        $eligibility = $eligibilityService->getEligibility($testCenter, $delivery);
+
+        if (!$eligibility) {
+            throw new common_exception_NotFound(__('Eligibility not found for provided search parameters.'));
```

Comment:
Please use `use` statements with `as` for these ugly class names.
=> NotFoundException
(see comment above)